JMscript.js

```diff
@@ -1,11 +1,11 @@
-console.dir(document);
-console.log('me');
+//console.dir(document);
+//console.log('me');
```

Comment:
Remove commented out code or have a comment explaining why it will be used later